e2e/testsuite/testsuite.go

```diff
@@ -42,7 +42,7 @@ const (
 	// ChainBRelayerName is the name given to the relayer wallet on ChainB
 	ChainBRelayerName = "rlyB"
 	// DefaultGasValue is the default gas value used to configure tx.Factory
-	DefaultGasValue = 500_000_0000
+	DefaultGasValue = 100_000_00
```

Comment:
Not sure on reasoning around this value being updated to be honest. 

Would be good to understand it more, or check why it was necessary. We can circle back on this.

Reply:
We can play around with it and see maybe! Probably should be 10_000_000 as well :D lol